### PR TITLE
[167756788] AdColony - Android - SDK causes the app to crash when trying...

### PR DESCRIPTION
**Pull request by hkosacki**
... to show an interstitial of AdColony when zone type is set to V4VC

Changed supported versions of AdColony adapter.

### CODE REVIEW

File: android-adapters/7.1.0/adapters.info [hunk 1]
```diff
@@ -0,0 +1,102 @@
+{
+   "adapters":[
+      {
+         "qualifiedName":"com.sponsorpay.mediation.AdColonyMediationAdapter",
+         "versions":[
+            "1.1.1"
```
Comment (gabriel-fortin) [inline android-adapters/7.1.0/adapters.info]: should be "1.1.3", right?

Comment (hkosacki) [inline android-adapters/7.1.0/adapters.info]: fixed

**Comment by hkosacki** [comment]
typos fixed

**Comment by gabriel-fortin** [comment]
review OK

**Comment by hkosacki** [comment]
PR rebased.

**Pull request merged**